pandas/_libs/tslibs/nattype.pyx

```diff
@@ -240,7 +241,8 @@ cdef class _NaT(datetime):
                 result = np.empty(other.shape, dtype="timedelta64[ns]")
                 result.fill("NaT")
                 return result
-        # other cases are same, swap operands is allowed even though we subtract because this is NaT
+        """" other cases are same, swap operands is allowed even though we subtract 
+        because this is NaT"""
```

Comment:
keep this as a comment, you can split it across multiple lines, e.g.
```
        # other cases are same, swap operands is allowed even
        # though we subtract because this is NaT
```